sdk/textanalytics/perf-tests/text-analytics/test/detectLanguage.spec.ts

```diff
@@ -4,12 +4,12 @@
 import { PerfTest, PerfOptionDictionary, getEnvVar } from "@azure/test-utils-perf";
 import {
   AzureKeyCredential,
-  TextAnalyticsClient,
-  DetectLanguageOptions,
+  TextAnalysisClient,
```

Comment:
Are we going to change TextAnalytics to TextAnalysis?

Reply:
This is a topic of interest among the crew and it has active discussion in https://github.com/Azure/azure-sdk-for-js/pull/21417#discussion_r853293304. Let's share our thoughts there.